org.eclipse.xpect.releng/target-platforms/eclipse_2024_06-xtext_nightly/org.eclipse.xpect.target.eclipse_2023_09-xtext_nightly.target

```diff
@@ -24,7 +24,7 @@
 <unit id="org.eclipse.jdt.feature.group" version="0.0.0"/>
 <unit id="org.eclipse.platform.feature.group" version="0.0.0"/>
 <unit id="org.eclipse.pde.feature.group" version="0.0.0"/>
-<repository location="https://download.eclipse.org/eclipse/updates/4.33-I-builds"/>
+<repository location="https://download.eclipse.org/eclipse/updates/4.33/"/>
```

Comment:
i assume correct value should be 4.24-I-builds. had already prepared this locally last week

Reply:
maybe whilst at also do the https://download.eclipse.org/releases/2024-12/ bump

Reply:
created https://github.com/eclipse/Xpect/pull/360/files
hope the permission sync with github runs soon, so that i can merge again